src/parser.ts

```diff
@@ -0,0 +1,38 @@
+import { AsyncAPIDocument, parse, parseFromUrl } from '@asyncapi/parser';
+import { isUrl, readDocument } from './utils';
+
+export interface ParsedOutput {
+  parsedDocument1: AsyncAPIDocument;
+  parsedDocument2: AsyncAPIDocument;
```

Comment:
Please don't use the numeric suffix for fields. Better will be:

 ```suggestion
  firstDocument: AsyncAPIDocument;
  secondDocument: AsyncAPIDocument;
```